scrapy_price_monitor/price_monitor/pipelines.py

```
@@ -1,21 +1,28 @@
-# -*- coding: utf-8 -*-
 from price_monitor import settings
-from hubstorage import HubstorageClient
+from price_monitor.collection_helper import CollectionHelper
 from price_monitor.utils import reversed_timestamp, get_product_names
 
 
-class CollectionStoragePipeline(object):
-
+class CollectionStoragePipeline:
     def open_spider(self, spider):
-        client = HubstorageClient(auth=settings.SHUB_KEY)
-        project = client.get_project(settings.SHUB_PROJ_ID)
         self.data_stores = {}
         for product_name in get_product_names():
-            self.data_stores[product_name] = project.collections.new_store(product_name)
+            store = CollectionHelper(
+                proj_id=settings.SHUB_PROJ_ID,
```

need to highlihgt in blog post that this needs to be customized

---

I'm not sure how it works, I didn't change it and it worked fine, project 999999 doesn't exist now, but I wonder if it will at some point, maybe better to leave it blank just in case and raise some error if project id is not defined?